### PR TITLE
Update shellrb.yml

### PR DESCRIPTION
**Pull request by faraazahmad**
Update the tag from help wanted to up for grabs

### CODE REVIEW

File: _data/projects/shellrb.yml [hunk 1]
```diff
@@ -8,5 +8,5 @@ tags:
 - Shell
 
 upforgrabs:
-  name: Help Wanted
+  name: Up For Grabs
   link: https://github.com/faraazahmad/shellrb/labels/help%20wanted
```
Comment (shiftkey) [inline _data/projects/shellrb.yml]: The "help wanted" label on the issue tracker still exists. Is this intentional, or should it also be renamed to "up for grabs"?

Comment (faraazahmad) [reply _data/projects/shellrb.yml]: I'm renaming all to up for grabs

Comment (shiftkey) [reply _data/projects/shellrb.yml]: 👍 then you'll need to change the `link` value below when that's been updated on the issue tracker

**Comment by shiftkey** [comment]
@faraazahmad thanks!

**Pull request merged**